src/boot/efi/cpio.c

```diff
@@ -312,6 +312,7 @@ EFI_STATUS pack_cpio(
                 UINT32 dir_mode,
                 UINT32 access_mode,
                 UINT32 tpm_pcr,
+                UINT32 tpm_pcr2,
```

Comment:
Any chance of using a list/array here? Aesthetically speaking it would be a bit nicer than hardcoding one and two